SpatialGDK/Documentation/content/get-started/gdk-template.md

```diff
@@ -99,20 +133,34 @@ To launch a cloud deployment, you need to prepare your server-worker and client-
 ##### Step 1: Set up your SpatialOS project name. 
 When you signed up for SpatialOS, your account was automatically associated with an organisation and a project, both of which have the same generated name.
 
-1. Find this name by going to the [Console (SpatialOS Documentation)](https://console.improbable.io). 
+1. Find this name by going to the [Console](https://console.improbable.io). 
 The name should look something like `beta_randomword_anotherword_randomnumber`. In the example below, it’s `beta_yankee_hawaii_621`. <br/>![Toolbar]({{assetRoot}}assets/set-up-template/template-project-page.png)<br/>_Image: The SpatialOS Console with a project name highlighted._
 2. In File Explorer, navigate to the `<YourProject>/spatial` directory and open the `spatialos.json` file in a text editor of your choice.
 3. Replace the `name` field with the project name shown in the Console. This tells SpatialOS which SpatialOS project you intend to upload to.
 
 ##### Step 2: Build your worker assemblies
 
-An assembly is what’s created when you run `BuildWorker.bat`. Assemblies are `.zip` files that contain all the files that your game uses when running in the cloud.
+Workers are the programs that connect to a SpatialOS world. In general, you use server-worker instances simulate the world, and players use client-worker instances to connect to your game. Worker assemblies are `.zip` files that contain all the files that your game uses when running in the cloud.
```

Comment:
I think "connect" is confusing as that suggests they are networking when they aren't really about networking.
By saying "you use" and then "players use", it's a bit confusing as to who is doing what. As you (devs) use in terms of doing development but players use in terms of playing a game. 
Also, we are saying servers simulate a world but players connect to a game. The game and world are the same thing here, so we should use the same word.

```suggestion
Workers are the programs that compute a SpatialOS world. In general, server-worker instances simulate the world, while game players connect to the world through client-worker instances. Worker assemblies are `.zip` files that contain built-out workers; that is all the files with compiled code that your game uses for running in the cloud.
```